recipes/tng/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+from conans import ConanFile, tools, CMake
+import os
+import glob
+
+
+class tngConan(ConanFile):
+    name = "tng"
+    description = "External GROMACS library for loading tng files."
+    license = "BSD-3-Clause"
+    topics = ("conan", "tng", "gromacs")
+    homepage = "https://github.com/gromacs/tng/"
```

Comment:
you'll need to point this ti gitlab as well =)

Reply:
Fixed :)